lib/locale.ts

```diff
@@ -3,11 +3,41 @@
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
 
+declare global {
+	// eslint-disable-next-line camelcase, no-var
+	var _nc_l10n_locale: string | undefined
+	// eslint-disable-next-line camelcase, no-var
+	var _nc_l10n_language: string | undefined
+}
```

Comment:
This will make it available also in exported typings.
Should probably be moved to a `globals.d.ts` file